src/sentry/seer/utils.py

```diff
@@ -44,3 +44,43 @@ def detect_breakpoints(breakpoint_request) -> BreakpointResponse:
         headers={"content-type": "application/json;charset=utf-8"},
     )
     return json.loads(response.data)
+
+
+# TODO: change these to NotRequired fields once Python version is 3.11
+class SimilarIssuesEmbeddingsRequestNotRequired(TypedDict, total=False):
```

Comment:
this is a workaround to get non-required fields in a `TypedDict` where the python version < 3.11

Reply:
you don't need to clarify this -- we use `total=False` TypedDicts all over the codebase